cmd/wharf/aggregator.go

```diff
@@ -24,18 +20,18 @@ kill endpoint.`,
 			return err
 		}
 
-		restConfig, ns, err := loadKubeconfig(aggregatorFlags.k8sOverrides)
+		restConfig, ns, err := loadKubeconfig()
 		if err != nil {
 			return err
 		}
 		aggregatorFlags.restConfig = restConfig
-		aggregatorFlags.namespace = ns
+		rootConfig.K8s.Namespace = ns
 		return nil
 	},
 }
 
 func init() {
 	rootCmd.AddCommand(aggregatorCmd)
 
-	addKubernetesFlags(aggregatorCmd.PersistentFlags(), &aggregatorFlags.k8sOverrides)
+	addKubernetesFlags(aggregatorCmd.PersistentFlags(), &rootConfig.K8s.Namespace)
```

Comment:
All the code paths seem to use `rootConfig.K8s.Namespace`. Suggest inlining it completely instead into `addKubernetesFlags` and `loadKubeconfig`.